src/Sylius/Bundle/ApiBundle/Resources/config/services/state_providers.xml

```diff
@@ -49,5 +49,14 @@
             <argument type="service" id="Sylius\Bundle\CoreBundle\SectionResolver\SectionProviderInterface" />
             <tag name="api_platform.state_provider" priority="10" />
         </service>
+
+        <service id="sylius_api.state_provider.shop.product_attributes" class="Sylius\Bundle\ApiBundle\StateProvider\ProductAttributesProvider">
```

Comment:
The `shop` part should not be necessary, as the provider is not in the `Shop` namespace
```suggestion
        <service id="sylius_api.state_provider.product_attributes" class="Sylius\Bundle\ApiBundle\StateProvider\ProductAttributesProvider">
```